cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedDeclarationEntry.qll

```diff
@@ -152,7 +156,13 @@ class TranslatedStaticLocalVariableDeclarationEntry extends TranslatedDeclaratio
     kind instanceof GotoEdge
   }
 
-  final override Instruction getInstructionSuccessor(InstructionTag tag, EdgeKind kind) {
+  final override Instruction getLastInstructionInternal() {
```

Comment:
Since this predicate can return multiple results: Should it be called `getALastInstructionInternal`? I think we should also add QLDoc to both this predicate (in `TranslatedElement.qll`) and `getLastChild`. Specifically, I'd like to know what "last" actually means. Is it "the instruction/child to be executed by this `TranslatedElement`"? Or something else?

Reply:
It's "an instruction/child for which the successor instruction is outside this `TranslatedElement`". I'll add a comment.